packages/url_launcher/url_launcher_web/lib/url_launcher_web.dart

```diff
@@ -8,6 +8,7 @@ import 'package:url_launcher_platform_interface/url_launcher_platform_interface.
 import 'package:platform_detect/platform_detect.dart' show browser;
 
 const _mailtoScheme = 'mailto';
+const _phoneScheme = 'tel';
```

Comment:
Can you create a Set containing this two URLs?

There was a bug with Safari in iOS where people would return to a "blank" page after sending an email (see https://github.com/flutter/flutter/issues/51461)

I *think* you'll need to extend the _isMailtoScheme method to also handle `tel`, and having them in a set is much easier. It'd be very easy to add `sms` at the same time as well.

You can also use Set methods to create the _supportedSchemes Set.

Reply:
Here's a link with the "Set" idea:

https://github.com/tugorez/plugins/pull/1

(there's other stuff that is not needed anymore, look at the `_safariTargetTopSchemes` Set and how it's used throughout the code)

Reply:
Seems awesome! will add it with the current code structure

Reply:
Just have one question though: 

Why is `_supportedSchemes` a private static variable of the class, but the `_safariTargetTopSchemes` a global private constant?

In the codebase that you shared this wasn't the case @ditman  🤔

Reply:
My version was just a sketch meant to give an idea (twice now! :P)

I think the current version is better:

`_safariTargetTopSchemes` is a property of an external entity to the plugin (Safari), it is defined in the same file, but it could come from some configuration file or some other package.
`_supportedSchemes` is a property of the plugin. What urls does this plugin support? And it makes sense it lives within the class.

(But you could argue that it can also be considered configuration, and I would also agree with you :P)